src/components/Navbar.jsx

```diff
@@ -99,7 +99,13 @@ const Navbar = () => {
                   }
                 ></div>
               </li>
-
+            </ul>
+            <div className="nav-buttons">
+              <div className="nav-item home">
+                <Button size="sm" className=" nav_signup_btn" to="/user/login">
```

Comment:
The sign in button should also Open the modal, but the modal's header should be `Sign In!` instead of `Sign Up!` for the button.

Also, can we have an `outlined variant` instead of a solid for this button?

Reply:
Hey have a question for this 
Which buttons should be present on the sign in modal, and if its same as signup should two buttons redirect to the same page?![image](https://github.com/IAmTamal/Milan/assets/29349136/ae3e966e-159b-4af4-98d6-8403f68a1510)


yes its possible to make outlined variant